asv/util.py

```diff
@@ -566,7 +566,7 @@ def _fix_env(s):
         debug_log = DebugLogBuffer(log)
         dots = False
     else:
-        debug_log = lambda c: None
+        debug_log = lambda c: None # noqa
```

Comment:
We shouldn't have bare `noqa`, we should always specify the error we want to ignore, and why.

In this case, why we don't fix the error properly?

Reply:
You are right. I was checking the other changes first, and I'm going to fix them.